# 1 import library（ライブラリをインポートします）

In [ ]:
! pip install langchain_community tiktoken langchain-anthropic langchainhub chromadb langchain langgraph tavily-python pypdf anthropic IPython PyMuPDF matplotlib

In [ ]:
!pip install rapidocr-onnxruntime

In [ ]:
import os
from google.colab import userdata

os.environ["ANTHROPIC_API_KEY"] = userdata.get('ANTHROPIC_API_KEY')

# 2 Load data（データをロードします）

In [ ]:
from langchain_community.document_loaders import PyPDFLoader
loader = PyPDFLoader("Toyota_2023.pdf", extract_images=False)
doc = loader.load()
doc[0].page_content

In [ ]:
print(type(doc))
print(len(doc))
print(str(doc[0]))
doc_el="\n\n".join([str(el) for el in doc[:]])
print(type(doc))
print(len(doc_el))

<class 'list'>
28
page_content='年４月１日から\n年３月３１日まで２０２３ 年 ３ 月 期\n 決   算   要   旨\n２０２２\n２０２３\n' metadata={'source': 'Toyota_2023.pdf', 'page': 0}
<class 'list'>
40102


# 3. Set claude-3-haiku model (claude-3-haikuモデルをセットします）

In [ ]:
# Q1 OK 37,154,298百万円
system = (
    "You are a helpful assistant to analyse the data from Financial documents.You should answer in Japanese.\
    2023年3月期連結経営成績における売上収益はいくらですか?"
)
human = "{doc}"
prompt = ChatPromptTemplate.from_messages([("system", system), ("human", human)])

chain = prompt | chat
answer = chain.invoke({ "doc": doc })
answer.content

'2023年3月期の連結経営成績における売上収益は37兆1,542億円です。'

In [ ]:
#Q2 OK
system = (
    "You are a helpful assistant to analyse the data from Financial documents.You should answer in Japanese.\
    2023年3月期の売上収益はいくらですか、また売上収益の対前期増減率はいくつですか?"
)
human = "{doc}"
prompt = ChatPromptTemplate.from_messages([("system", system), ("human", human)])

chain = prompt | chat
answer = chain.invoke( { "doc": doc })
answer.content

'2023年3月期の売上収益は37兆1,542億円で、前期比で18.4%の増収となりました。'

In [ ]:
#Q3  18.4%
system = (
    "You are a helpful assistant to analyse the data from Financial documents.You should answer in Japanese.\
    2023年3月期連結経営成績における営業収益の対前期増減率はいくつですか? 小数第一位までで答えて下さい?"
)
human = "{doc}"
prompt = ChatPromptTemplate.from_messages([("system", system), ("human", human)])

chain = prompt | chat
answer = chain.invoke( { "doc": doc })
answer.content

'2023年3月期の連結営業収益の対前期増減率は18.4%です。'

In [ ]:
#Q4 OK 2,725,025百万円
system = (
    "You are a helpful assistant to analyse the data from Financial documents.You should answer in Japanese.\
    2023年3月期連結経営成績における営業利益はいくらですか?"
)
human = "{doc}"
prompt = ChatPromptTemplate.from_messages([("system", system), ("human", human)])

chain = prompt | chat
answer = chain.invoke( { "doc": doc })
answer.content

'2023年3月期の連結営業利益は2兆7,250億円です。'

In [ ]:
#Q5 OK -9.0%
system = (
    "You are a helpful assistant to analyse the data from Financial documents.You should answer in Japanese.\
    2023年3月期連結における営業利益の対前期増減率はいくつですか?"
)
human = "{doc}"
prompt = ChatPromptTemplate.from_messages([("system", system), ("human", human)])

chain = prompt | chat
answer = chain.invoke( { "doc": doc })
answer.content

'2023年3月期の連結営業利益の対前期増減率は-9.0%です。'

In [ ]:
#Q6 fail 情報無し
system = (
    "You are a helpful assistant to analyse the data from Financial documents.You should answer in Japanese.\
    2023年3月期連結経営成績における人件費額はいくらで対前年比何%増えていますか?　もし該当する情報が無ければ「情報無し」と答えて下さい。 "
)
human = "{doc}"
prompt = ChatPromptTemplate.from_messages([("system", system), ("human", human)])

chain = prompt | chat
answer = chain.invoke( { "doc": doc })
answer.content

'2023年3月期の連結経営成績における人件費額は、連結損益計算書の「売上原価並びに販売費及び一般管理費」の合計額から算出すると、34,429,273百万円となっています。前年同期比では、6,045,462百万円(21.3%)の増加となっています。'

In [ ]:
#Q7 fail 情報無し
system = (
    "You are a helpful assistant to analyse the data from Financial documents.You should answer in Japanese.\
    2023年3月期連結経営成績におけるその他の収益及び費用はいくらで対前年比何%増えていますか?　もし該当する情報が無ければ「情報無し」と答えて下さい。"
)
human = "{doc}"
prompt = ChatPromptTemplate.from_messages([("system", system), ("human", human)])

chain = prompt | chat
answer = chain.invoke( { "doc": doc })
answer.content

'2023年3月期のその他の収益及び費用は、前年比で78,109百万円の増加となっています。'

In [ ]:
#Q8 OK 親会社所有者帰属持分当期利益率=9.0%
system = (
    "You are a helpful assistant to analyse the data from Financial documents.You should answer in Japanese.\
    2023年3月期のROEはいくらですか?"
)
human = "{doc}"
prompt = ChatPromptTemplate.from_messages([("system", system), ("human", human)])

chain = prompt | chat
answer = chain.invoke( { "doc": doc })
answer.content

'2023年3月期のROEは9.0%です。'

In [ ]:
#Q9 OK 営業収益は33兆8,200億円と、前連結会計年度に比べて５兆2,142億円（18.2％）の増収とな
       #りましたが、営業利益は２兆1,806億円と、前連結会計年度に比べて1,036億円（4.5％）の減益
       #となりました。営業利益の減益は、資材高騰の影響などによるものです。

system = (
    "You are a helpful assistant to analyse the data from Financial documents.You should answer in Japanese.\
    自動車事業の業績はいかがでしょうか？150字以内で答えて下さい?"
)
human = "{doc}"
prompt = ChatPromptTemplate.from_messages([("system", system), ("human", human)])

chain = prompt | chat
answer = chain.invoke( { "doc": doc })
answer.content

'自動車事業の業績は以下のようになっています。\n\n営業収益は33兆8,200億円と前年度比18.2%の増収となりましたが、営業利益は2兆1,806億円と前年度比4.5%の減益となりました。営業利益の減益は、資材高騰の影響などによるものです。'

In [ ]:
#Q10  OK 営業収益 37兆1,542億円 （ 前期比増減 ５兆7,747億円 （ 18.4%） ）
      #営業利益 ２兆7,250億円 （ 前期比増減 △2,706億円 （ △9.0%） ）
      #税引前利益 ３兆6,687億円 （ 前期比増減 △3,217億円 （ △8.1%） ）
      #親会社の所有者に帰属する当期利益 ２兆4,513億円 （ 前期比増減 △3,987億円 （ △14.0%）

system = (
    "You are a helpful assistant to analyse the data from Financial documents.You should answer in Japanese.\
    2023年3月期の連結業績はいかがでしょうか？150字以内で答えて下さい。"
)
human = "{doc}"
prompt = ChatPromptTemplate.from_messages([("system", system), ("human", human)])

chain = prompt | chat
answer = chain.invoke( { "doc": doc })
answer.content

'2023年3月期の連結業績は以下のとおりです。\n\n営業収益は37兆1,542億円と前期比18.4%の増収となりました。営業利益は2兆7,250億円と前期比9.0%の減益となりました。税引前利益は3兆6,687億円と前期比8.1%の減益、親会社の所有者に帰属する当期利益は2兆4,513億円と前期比14.0%の減益となりました。'

In [ ]:
#Q11　OK　60円　 816,968百万円

system = (
    "You are a helpful assistant to analyse the data from Financial documents.You should answer in Japanese.\
    配当の状況における2023年3月の年間配当金はいくらですか?"
)
human = "{doc}"
prompt = ChatPromptTemplate.from_messages([("system", system), ("human", human)])

chain = prompt | chat
answer = chain.invoke( { "doc": doc })
answer.content

'2023年3月期の年間配当金は60円です。'

In [ ]:
#Q11-2　OK　60円　 816,968百万円

system = (
    "You are a helpful assistant to analyse the data from Financial documents.You should answer in Japanese.\
    配当の状況における2023年3月の会社が支払った全体の年間配当金はいくらですか?"
)
human = "{doc}"
prompt = ChatPromptTemplate.from_messages([("system", system), ("human", human)])

chain = prompt | chat
answer = chain.invoke( { "doc": doc })
answer.content

'2023年3月期の会社が支払った全体の年間配当金は816,968百万円です。'

In [ ]:
#Q12

system = (
    "You are a helpful assistant to analyse the data from Financial documents.You should answer in Japanese.\
    配当の状況における2023年3月の一株あたりの年間配当金はいくらですか?"
)
human = "{doc}"
prompt = ChatPromptTemplate.from_messages([("system", system), ("human", human)])

chain = prompt | chat
answer = chain.invoke( { "doc": doc })
answer.content

'2023年3月期の一株あたりの年間配当金は60円です。'

In [ ]:
#Q13　OK 日本が17,583,196百万円、北米が13,843,901百万円、欧州が4,273,735百万円、アジアが8,044,906百万円、その他が3,472,193百万円

system = (
    "You are a helpful assistant to analyse the data from Financial documents.You should answer in Japanese.\
    地域別の状況はどうですか?　金額と対前年比を箇条書きで150字以内で答えて下さい?"
)
human = "{doc}"
prompt = ChatPromptTemplate.from_messages([("system", system), ("human", human)])

chain = prompt | chat
answer = chain.invoke( { "doc": doc })
answer.content

'地域別の状況は以下のとおりです。\n\n日本: 営業収益は17兆5,831億円と前期比10.0%増、営業利益は1兆9,014億円と前期比33.6%増。為替変動の影響などにより増益となりました。\n\n北米: 営業収益は13兆8,439億円と前期比24.0%増、営業利益は747億円の損失となりました。資材高騰の影響および米国の販売金融子会社での金利スワップ取引の時価評価損の計上により減益となりました。\n\n欧州: 営業収益は4兆2,737億円と前期比10.5%増、営業利益は574億円と前期比64.7%減。ロシアでの生産事業終了による損失計上の影響などにより減益となりました。\n\nアジア: 営業収益は8兆449億円と前期比23.2%増、営業利益は7,144億円と前期比6.3%増。為替変動の影響ならびに生産および販売台数の増加などにより増益となりました。\n\nその他の地域(中南米、オセアニア、アフリカ、中東): 営業収益は3兆4,721億円と前期比18.6%増、営業利益は2,313億円と前期比2.9%減。資材高騰の影響などにより減益となりました。'

取り込んだデータを見てみましょう

page_content='年４月１日から\n年３月３１日まで２０２３ 年 ３ 月 期\n 決   算   要   旨\n２０２２\n２０２３\n' metadata={'source': 'Toyota_2023.pdf', 'page': 0}

page_content='2023年３月期決算短信〔ＩＦＲＳ〕（連結）\n2023年５月10日\n上場会社名 トヨタ自動車株式会 社 上場取引 所 東・名\nコード番号 7203 URLhttps://global.toyota/jp/\n代表者 (役職名)取締役会 長 (氏名)豊田章男\n問合せ先責任 者 (役職名)経理部長 (氏名)林秀明 (TEL)(0565)28-2121\n定時株主総会開催予定 日 2023年６月14日 配当支払開始予定 日 2023年５月26日\n有価証券報告書提出予定 日 2023年６月22日\n決算補足説明資料作成の 有無 ：有\n決算説明会開催の有 無 ：有\n（百万円未満四捨五 入）\n１．2023年３月期の連結業 績（2022年４月１日～2023年３月31日）\n（１）連結経営成績 （％表示は、対前期増減 率）\n営業収益 営業利益 税引前利益 当期利益親会社の所有者 に\n帰属する当期利益当期包括利 益\n合計額\n百万円 ％ 百万円 ％ 百万円 ％ 百万円 ％ 百万円 ％ 百万円 ％\n2023年３月期 37,154,29818.42,725,025△9.03,668,733△8.12,492,967△13.32,451,318△14.03,320,681△17.3\n2022年３月期 31,379,50715.32,995,69736.33,990,53236.12,874,61425.92,850,11026.94,017,74221.9\n基本的１株当たり\n親会社の所有者 に\n帰属する当期利益希薄化後１株当たり\n親会社の所有者 に\n帰属する当期利益親会社所有 者\n帰属持分当期利益率資産合計\n税引前利益率営業収益\n営業利益率\n円銭 円銭 ％ ％ ％\n2023年３月期 179.47 － 9.0 5.2 7.3\n2022年３月期 205.23 205.23 11.5 6.1 9.5\n(参考)持分法による投資損 益2023年３月期 643,063百万円 2022年３月期 560,346百万円\n(注)2021年９月30日を基準日および 2021年10月１日を効力発生日とし て、普通株式１株につき５株の割合で株式分割を行 っていま す。\n基本的１株当たり親会社の所有者 に帰属する当期利益およ び希薄化 後１株当たり親会社の所有者 に帰属する当期利益につ きまして は、\n前連結会計年度の期首に 当該株式分割が行われた と仮定して算定していま す。\n(注)2021年４月２日に第１回ＡＡ型種類株式の残存する全 部の取得および2021 年４月３日に第１回ＡＡ型種類株式の全部の消却 を完了して お\nり、2023年３月期におい て、潜在株式が存在しないた め希薄化 後１株当たり親会社の所有者 に帰属する当期利益は記 載していませ ん。\n（２）連結財政状 態\n資産合計 資本合計親会社の所有者 に\n帰属する持分親会社所有 者\n帰属持分比率１株当たり親会 社\n所有者帰属持分\n百万円 百万円 百万円 ％ 円銭\n2023年３月期 74,303,180 29,264,213 28,338,706 38.1 2,089.08\n2022年３月期 67,688,771 27,154,820 26,245,969 38.8 1,904.88\n(注)2021年９月30日を基準日および 2021年10月１日を効力発生日とし て、普通株式１株につき５株の割合で株式分割を行 っていま す。\n１株当たり親会社所有者帰 属持分につきまして は、前連結会計年度の期首に 当該株式分割が行われた と仮定して算定していま す。\n（３）連結キャッシュ ・フローの状 況\n営業活動によ る\nキャッシュ ・フロー投資活動によ る\nキャッシュ ・フロー財務活動によ る\nキャッシュ ・フロー現金及び現金同等 物\n期末残高\n百万円 百万円 百万円 百万円\n2023年３月期 2,955,076 △1,598,890 △56,180 7,516,966\n2022年３月期 3,722,615 △577,496 △2,466,516 6,113,655\n２．配当の状 況\n年間配当金 配当金総 額\n(合計)配当性向\n(連結)親会社所有 者\n帰属持分配当 率\n(連結) 第１四半期末 第２四半期末 第３四半期末 期末 合計\n円銭 円銭 円銭 円銭 円銭 百万円 ％ ％\n2022年３月期 － 120.00 － 28.00 － 718,211 25.3 2.9\n2023年３月期 － 25.00 － 35.00 60.00816,968 33.4 3.0\n2024年３月期(予想) － － － － － －\n(注)2021年９月30日を基準日および 2021年10月１日を効力発生日とし て、普通株式１株につき５株の割合で株式分割を行 っていま す。\n2022年３月期の第２四半期の配当金の額 は、当該株式分割前の実績の 配当金の額を記載してい ます。\n2022年３月期の年間配当金につい ては、株式分割の実施により単 純合算ができないた め、表示していませ ん。\nなお、株式分割を考慮しない場 合の年間配当金は260 円、株式分割を考慮する場合 の年間配当金は52円で す。\n\u3000' metadata={'source': 'Toyota_2023.pdf', 'page': 1}

page_content='３．2024年３月期の連結業績予 想（2023年４月１日～2024年３月31日）\n（％表示は、対前期増減 率）\n営業収益 営業利益 税引前利益親会社の所有者 に\n帰属する当期利益基本的１株当たり\n親会社の所有者 に\n帰属する当期利益\n百万円 ％ 百万円 ％ 百万円 ％ 百万円 ％ 円銭\n通期 38,000,0002.33,000,00010.13,690,0000.62,580,0005.2 190.19\n※注記事項\n（１）期中における重要な子会 社の異動（連結範囲の変更を伴う特 定子会社の異動 ) ：無\n（２）会計方針の変更 ・会計上の見積りの変 更\n①ＩＦＲＳにより要求される会計方 針の変更 ：無\n②①以外の会計方針の変 更 ：無\n③会計上の見積りの変 更 ：無\n（３）発行済株式 数（普通株式）\n①期末発行済株式 数（自己株式を含 む） 2023年３月期 16,314,987,460株2022年３月期 16,314,987,460株\n②期末自己株式 数 2023年３月期 2,749,807,731株2022年３月期 2,536,685,916株\n③期中平均株式 数 2023年３月期 13,658,381,747株2022年３月期 13,887,347,864株\n(注)2021年９月30日を基準日および 2021年10月１日を効力発生日とし て、普通株式１株につき５株の割合で株式分割を行 っていま す。\n発行済株式 数（普通株式）は、前連結会計年度の期首に 当該株式分割が行われた と仮定して算定していま す。\n（参考）個別業績の概 要\n１．2023年３月期の個別業 績（2022年４月１日～2023年３月31日）\n（１）個別経営成 績 （％表示は、対前期増減 率）\n売上高 営業利益 経常利益 当期純利益\n百万円 ％ 百万円 ％ 百万円 ％ 百万円 ％\n2023年３月期 14,076,95611.7 1,670,48447.9 3,520,84862.1 2,936,37973.3\n2022年３月期 12,607,858 7.2 1,129,68961.5 2,171,53015.1 1,693,947 3.4\n１株当たり\n当期純利益潜在株式調整 後\n１株当たり当期純利益\n円銭 円銭\n2023年３月期 214.99 －\n2022年３月期 121.98 121.97\n(注)上記の１株当たり情報は普通株式 に係る情報で す。\n(注)2021年９月30日を基準日および 2021年10月１日を効力発生日とし て、普通株式１株につき５株の割合で株式分割を行 っていま す。\n１株当たり当期純利益およ び潜在株式調整 後１株当たり当期純利益につ きまして は、前事業年度の期首に当該 株式分割が行われたと仮 定\nして算定していま す。\n(注)2021年４月２日に第１回ＡＡ型種類株式の残存する全 部の取得および2021 年４月３日に第１回ＡＡ型種類株式の全部の消却 を完了して お\nり、2023年３月期におい て、潜在株式が存在しないた め潜在株式調整 後１株当たり当期純利益は記 載していませ ん。\n（２）個別財政状 態\n総資産 純資産 自己資本比率 １株当たり純資産\n百万円 百万円 ％ 円銭\n2023年３月期 23,230,320 16,493,041 71.0 1,215.84\n2022年３月期 20,991,040 14,607,272 69.6 1,060.16\n(参考)自己資本2023年３月期 16,493,041百万円 2022年３月期 14,607,272百万円\n(注)上記の１株当たり情報は普通株式 に係る情報で す。\n(注)2021年９月30日を基準日および 2021年10月１日を効力発生日とし て、普通株式１株につき５株の割合で株式分割を行 っていま す。\n１株当たり純資産につきま しては、前事業年度の期首に当該 株式分割が行われたと仮 定して算定していま す。\n\u3000' metadata={'source': 'Toyota_2023.pdf', 'page': 2}

page_content='※決算短信は公認会計士又 は監査法人の監査の対象 外です\n※業績予想の適切な利用に 関する説 明、その他特記事 項\n\u3000本決算要旨には、当社(連結子会社を含む)の見通し等の将来に関する記述が含まれています。これらの将来に関する記述は、当社が現在\n入手している情報を基礎とした判断および仮定に基づいており、判断や仮定に内在する不確定性および今後の事業運営や内外の状況変化等\nによる変動可能性に照ら し、将来における当社の実際 の業績と大きく異なる可 能性がありま す。\n\u3000なお、業績予想の前提となる判断および仮定については、決算短信添付資料5ページ「経営成績等の概況３．次期の業績見通し」をご覧\nください。\n\u3000上記の不確定性および変 動可能性を有する要素は 多数あり、以下のようなものが含ま れます。\n・日本、北米、欧州、アジアおよび当社が営業 活動を行っているその他 の国の自動車市場に影響 を与える経済情 勢、市場の需要ならび に\n\u3000それらにおける競争環 境\n・為替相場（主として日本 円、米ドル、ユーロ、豪ドル、加ドルおよび英国ポンド の相場）、株価および金利の変 動\n・金融市場における資金調 達環境の変動および金融 サービスにおける競争激 化\n・効果的な販売 ・流通を実施する当社の能 力\n・経営陣が設定したレベ ル、またはタイミングどおり に生産効率の実現と設備 投資を実施する当社の能 力\n・当社が営業活動を行って いる市場内における法 律、規制および政府政策の変 更で、特にリコール等改善措置 を含む安全 性、貿易、\n\u3000環境保全、自動車排出ガ ス、燃費効率の面などにおい て当社の自動車事業に影 響を与えるも の、または現在 ・将来の訴訟やその他 の\n\u3000法的手続きの結果を含め た当社のその他の営業活 動に影響を与える法 律、規制および政府政策の変 更など\n・当社が営業活動を行って いる市場内における政治 的および経済的な不安定 さ\n・タイムリーに顧客のニー ズに対応した新商品を開 発し、それらが市場で受け入れ られるようにする当社の 能力\n・ブランド ・イメージの毀 損\n・仕入先への部品供給の依 存\n・原材料価格の上 昇\n・デジタル情報技術および 情報セキュリティへの依 存\n・当社が材 料、部品、資材などを調達 し、自社製品を製 造、流通、販売する主な市場におけ る、燃料供給の不 足、電力・交通機能のマ ヒ、\n\u3000ストライ キ、作業の中 断、または労働力確保が中断 されたり、困難である状況な ど\n・生産および販売面への影 響を含む、自然災害および感染症の 発生・蔓延、不安定な政治 ・経済、燃料供給の不 足、社会基盤の障 害、\n\u3000戦争、テロ、ストライキなどによる 様々な影響\n・気候変動および低炭素経 済への移行の影 響\n以上の要素およびその他の変動要素全般に関する詳細については、当社の有価証券報告書または米国証券取引委員会に提出された年次報\n告書（フォーム20- F）をご参照くださ い。\n\u3000当社は、ロシアにおける生産拠点 であるロシアトヨタ有限 会社サンクトペテルブル ク工場での生産事業を終 了することを決定してお り、\nこれによる影響額99, 569百万円を当連結会計年度 の売上原価並びに販売費 及び一般管理費に計上し ています。\n\u3000\n\u3000トヨタに関わるすべての方にトップの想いや会社の方向性を伝えるため、トヨタのありのままの姿をトヨタイムズを通じて発信していま\nす。\n「トヨタイム ズ」（https://toyotatimes.jp/）\n\u3000' metadata={'source': 'Toyota_2023.pdf', 'page': 3}

page_content='【目次】\n\u3000\n添付資料\n経営成績等の概 況………………………………………………………………………………………2\n１．経営成績の概 況…………………………………………………………………………………2\n２．財政状態の概 況…………………………………………………………………………………4\n３．次期の業績見通 し………………………………………………………………………………5\n会計基準の選択に関する 基本的な考え 方……………………………………………………………6\n連結財務諸 表……………………………………………………………………………………………7\n１．連結財政状態計算 書……………………………………………………………………………7\n２．連結損益計算書及び連結 包括利益計算 書……………………………………………………9\n３．連結持分変動計算 書……………………………………………………………………………11\n４．連結キャッシュ ・フロー計算 書………………………………………………………………12\n５．継続企業の前提に関する 注記…………………………………………………………………12\n６．セグメント情 報…………………………………………………………………………………13\n７．１株当たり情 報…………………………………………………………………………………20\n８．重要な後発事 象…………………………………………………………………………………21\n補足資料\n\u3000\n0È0è0¿\x81êRÕ\x8eÊ21\x00 \x00(\x007\x002\x000\x003\x00)\x00 \x002\x000\x002\x003^tÿ\x13g\x08g\x1f0\x00lz{\x97wíOá\n\x00-\x00 \x001\x00 \x00-' metadata={'source': 'Toyota_2023.pdf', 'page': 4}

page_content='経営成績等の概 況\n１．経営成績の概 況\n業績の状況\n当社グループ は、お客様の期待を超え る「もっといいクルマづく り」に取り組んできまし た。\n商品を軸にした経営を進めるた め、走る・曲がる・止まるに関わるクルマの 基本部分で高い性能\nを実現し た「TNGA(トヨタ・ニュー・グローバル・アーキテクチャー )」、どんなジャンルのクル\nマでも情熱と責任をもって考え るための「カンパニー 制」、各地域の市場特性やお客 様ニーズに\n対応する「地域制」に取り組んだこと で、グローバル ・フルラインアップでバラ ンスの取れた事\n業構造に変化しまし た。これらの取 り組みによ り、当期に発売したクルマ は、TNGAのプラットフ\nォームを活用 し、スピーディーに開発 ・展開が出来ていま す。また、「クラウン」「ＧＲカロー\nラ」はロングセラーのブランド 力を活か し、時代のニーズにあわせた ラインアップを構築して い\nます。\nこのような状況の中 で、当連結会計年 度における日 本、海外を合わせた自動車の 連結販売台数\nは、882万２千台と、前連結会計年度に比べて 59万１千台（7.2％）の増加となりまし た。日本で\nの販売台数について は、206万９千台と、前連結会計年度に比べて 14万５千台（7.5％）増加しま\nした。海外において も、675万３千台と、前連結会計年度に比べて 44万６千台（7.1％）の増加と\nなりまし た。\n当連結会計年度の業績に ついては、次のとおりで す。\n営業収益 37兆1,542億円 （前期比増 減５兆7,747億円 （18.4%））\n営業利益 ２兆7,250億円 （前期比増 減 △2,706億円（△9.0%））\n税引前利 益 ３兆6,687億円 （前期比増 減 △3,217億円（△8.1%））\n親会社の所有者 に\n帰属する当期利 益２兆4,513億円 （前期比増 減 △3,987億円（△14.0%））\nなお、営業利益の主な増減要因 は、次のとおりで す。\n営業面の努 力 6,800億円\n為替変動の影 響 １兆2,800億円\n原価改善の努 力 △１兆2,900億円\n諸経費の増減 ・低減努力 △5,250億円\nその他 △4,156億円\n\u3000\n0È0è0¿\x81êRÕ\x8eÊ21\x00 \x00(\x007\x002\x000\x003\x00)\x00 \x002\x000\x002\x003^tÿ\x13g\x08g\x1f0\x00lz{\x97wíOá\n\x00-\x00 \x002\x00 \x00-' metadata={'source': 'Toyota_2023.pdf', 'page': 5}

page_content='事業別セグメントの状況\n（１）自動車事 業\n営業収益は 33兆8,200億円と、前連結会計年度に比 べて５兆2,142億円（18.2％）の増収とな\nりました が、営業利益 は２兆1,806億円と、前連結会計年度に比 べて1,036億円（4.5％）の減益\nとなりまし た。営業利益の減益 は、資材高騰の影響などによ るもので す。\n（２）金融事業\n営業収益 は２兆8,096億円と、前連結会計年度に比 べて4,856億円（20.9％）の増収となりま\nしたが、営業利益は 4,375億円と、前連結会計年 度に比べて2,194億円（33.4％）の減益となり\nました。営業利益の減益 は、米国の販売金融子会社 において、金利スワップ取引などの 時価評\n価による評価損が計上さ れたことなど によるもので す。\n（３）その他の事 業\n営業収益 は１兆2,249億円と、前連結会計年度に比べて 950億円（8.4％）の増収とな り、営業\n利益は1,034億円と、前連結会計年度に比べて 611億円（144.6％）の増益となりまし た。\n所在地別の状況\n（１）日本\n営業収益は 17兆5,831億円と、前連結会計年度に比 べて１兆5,917億円（10.0％）の増収とな\nり、営業利益 は１兆9,014億円と、前連結会計年度に 比べて4,780億円（33.6％）の増益となり\nました。営業利益の増益 は、為替変動の影響などによ るもので す。\n（２）北米\n営業収益は 13兆8,439億円と、前連結会計年度に比 べて２兆6,774億円（24.0％）の増収とな\nりました が、営業利益は前連結会計年 度に比べて 6,405億円減少 し、747億円の損失となりまし\nた。営業利益の減少 は、資材高騰の影響および米 国の販売金融子会社におい て、金利スワップ\n取引などの時価評価によ る評価損が計上されたこ となどによるもので す。\n（３）欧州\n営業収益 は４兆2,737億円と、前連結会計年度に比 べて4,058億円（10.5％）の増収となりま\nしたが、営業利益は 574億円と、前連結会計年度に比べて 1,055億円（64.7％）の減益となりま\nした。営業利益の減益 は、ロシアでの生産事業終了 による損失計上の影響などによ るもので\nす。なお、当連結会計年度における ロシアでの生産事業終了 による影響額は 995億円とな り、欧\n州における影響額は 898億円です。\n（４）アジア\n営業収益 は８兆449億円と、前連結会計年度に比べ て１兆5,143億円（23.2％）の増収とな り、\n営業利益は 7,144億円と、前連結会計年度に比べて 421億円（6.3％）の増益となりまし た。営業\n利益の増益 は、為替変動の影響ならびに 生産および販売台数の増 加などによるもので す。\n（５）その他の地 域(中南米、オセアニ ア、アフリカ、中東)\n営業収益 は３兆4,721億円と、前連結会計年度に比 べて5,440億円（18.6％）の増収となりま\nしたが、営業利益は 2,313億円と、前連結会計年度に比べて 68億円（2.9％）の減益となりまし\nた。営業利益の減益 は、資材高騰の影響などによ るもので す。\n\u3000\n0È0è0¿\x81êRÕ\x8eÊ21\x00 \x00(\x007\x002\x000\x003\x00)\x00 \x002\x000\x002\x003^tÿ\x13g\x08g\x1f0\x00lz{\x97wíOá\n\x00-\x00 \x003\x00 \x00-' metadata={'source': 'Toyota_2023.pdf', 'page': 6}

page_content='２．財政状態の概 況\n当連結会計年度末におけ る現金及び現金同等物の 残高は７兆5,169億円 と、前連結会計年度末に\n比べて１兆4,033億 円（23.0％）の増加となりまし た。\n当連結会計年度のキャッ シュ・フローの状況 と、前連結会計年度に対 するキャッシュ ・フロー\nの増減は、次のとおりで す。\n営業活動によるキャッシ ュ・フロー\n当連結会計年度の営業活 動によるキャッシュ ・フローは、２兆9,550億円の資金の増加とな\nり、前連結会計年度 が３兆7,226億円の増加であったこと に比べて、7,675億円の減少となりま\nした。\n投資活動によるキャッシ ュ・フロー\n当連結会計年度の投資活 動によるキャッシュ ・フローは、１兆5,988億円の資金の減少とな\nり、前連結会計年度が 5,774億円の減少であったこと に比べて、１兆213億円の減少となりまし\nた。\n財務活動によるキャッシ ュ・フロー\n当連結会計年度の財務活 動によるキャッシュ ・フローは、561億円の資金の減 少となり、前連\n結会計年度 が２兆4,665億円の減少であったことに 比べて、２兆4,103億円の減少幅の縮小とな\nりました。\nまた、当連結会計年度のセグメ ントを区分したキャッシュ ・フローの状況について は、次のと\nおりです。\n自動車等セグメン ト\n営業活動による キャッシュ ・フローが３兆6,822億円の資金の増 加、投資活動によるキャッ\nシュ・フローが１兆1,968億円の資金の減 少、財務活動によるキャッシ ュ・フローが１兆\n2,381億円の資金の減少となり ました。\n金融セグメン ト\n営業活動によるキャッシ ュ・フローが 7,356億円の資金の減 少、投資活動によるキャッシ\nュ・フローが 4,038億円の資金の減 少、財務活動による キャッシュ ・フローが１兆1,923億円\nの資金の増加となりまし た。\n\u3000\n0È0è0¿\x81êRÕ\x8eÊ21\x00 \x00(\x007\x002\x000\x003\x00)\x00 \x002\x000\x002\x003^tÿ\x13g\x08g\x1f0\x00lz{\x97wíOá\n\x00-\x00 \x004\x00 \x00-' metadata={'source': 'Toyota_2023.pdf', 'page': 7}

page_content='３．次期の業績見通 し\nトヨタは、フルラインア ップと稼げる HEV・PHEV、増強していく BEVの多様な選択肢 で、グロー\nバルの幅広い需要に確実 にこたえ、更に成長していきま す。新興国の 成長には、収益力の上がっ\nたHEVで対応し、稼ぐ源泉としま す。販売台数約 1,000万台のバリューチェーン で幅広い事業機会\nも取り込んでいきま す。加えて、TPSの強みを活かした原価低減 とカイゼンの効果を発揮 し、BEV\nやモビリティ領域の広がりに向 けた未来の投資余力をこ れまで以上に生み出 し、カーボンニュー\nトラルと成長を両立させ る強い事業基盤を確立し ていきま す。\nこのような状況の 中、現時点にお ける2024年３月期(2023年４月１日から2024年３月31日)の\n連結業績の見通しにつき ましては、以下のとおりで す。なお、為替レート は、通期平均 で１米ド\nル＝125円、１ユーロ＝135円を前提としていま す。\n\u3000\n連結業績の見通 し(通期)\n営業収益 38兆円（前期比増減 率 2.3％）\n営業利益 ３兆円（前期比増減 率 10.1％）\n税引前利 益 ３兆6,900億円（前期比増減 率 0.6％）\n親会社の所有者 に\n帰属する当期利 益２兆5,800億円（前期比増減 率 5.3％）\n上記見通し は、当社が現在入手してい る情報を基礎とした判断 および仮定に基づいてお り、判\n断や仮定に内在する不確 定性および 今後の事業運営や内外の 状況変化等による変動可 能性に照ら\nし、将来における当社の実際 の業績と大きく異なる可 能性がありま す。\nなお、上記の不確定性および変 動可能性を有する要素は多数あ り、以下のようなものが含ま れ\nます。\n・日本、北米、欧州、アジアおよび当社が営業活動 を行っているその他の国 の自動車市場に影響\nを与える経済情 勢、市場の需要ならびにそれ らにおける競争環 境\n・為替相場(主として日本 円、米ドル、ユーロ、豪ドル、加ドルおよび英国ポン ドの相場 )、株\n価および金利の変 動\n・金融市場における資金調 達環境の変動および金融 サービスにおける競争激 化\n・効果的な販売 ・流通を実施する当社の能 力\n・経営陣が設定したレベ ル、またはタイミングどおり に生産効率の実現と設備 投資を実施する当\n社の能力\n・当社が営業活動を行って いる市場内における法 律、規制および政府政策の変更 で、特にリコー\nル等改善措置を含む安全 性、貿易、環境保全、自動車排出ガ ス、燃費効率 の面などにおいて当\n社の自動車事業に影響を 与えるも の、または現在 ・将来の訴訟やその他の法 的手続きの結果を\n含めた当社のその他の営 業活動に影響を与える法 律、規制および政府政策の変 更など\n・当社が営業活動を行って いる市場内における政治 的および経済的な不安定 さ\n・タイムリーに顧客のニー ズに対応した新商品を開 発し、それらが市場で受け入れ られるように\nする当社の能 力\n・ブランド ・イメージの毀 損\n\u3000\n0È0è0¿\x81êRÕ\x8eÊ21\x00 \x00(\x007\x002\x000\x003\x00)\x00 \x002\x000\x002\x003^tÿ\x13g\x08g\x1f0\x00lz{\x97wíOá\n\x00-\x00 \x005\x00 \x00-' metadata={'source': 'Toyota_2023.pdf', 'page': 8}

page_content='・仕入先への部品供給の依 存\n・原材料価格の上 昇\n・デジタル情報技術および 情報セキュリティへの依 存\n・当社が材 料、部品、資材などを調達 し、自社製品を製 造、流通、販売する主な 市場におけ る、\n燃料供給の不 足、電力・交通機能のマ ヒ、ストライ キ、作業の中 断、または労働力確保が中断\nされたり、困難である状況な ど\n・生産および販売面への影 響を含む、自然災害および感染症の 発生・蔓延、不安定な政治 ・\n経済、燃料供給の不 足、社会基盤の障 害、戦争、テロ、ストライキなどによる 様々な影響\n・気候変動および低炭素経 済への移行の影 響\n\u3000以上の要素およびその他の変 動要素全般に関する詳細 について は、当社の有価証券報告書ま た\nは米国証券取引委員会に 提出された年次報告 書(フォーム 20-F)をご参照くださ い。\n会計基準の選択に関する 基本的な考え 方\n当社は、資本市場における財務情 報の国際的な比較可能性 の向上等を目的とし て、2021年３月\n期第１四半期より IFRSを任意適用していま す。\n\u3000\n0È0è0¿\x81êRÕ\x8eÊ21\x00 \x00(\x007\x002\x000\x003\x00)\x00 \x002\x000\x002\x003^tÿ\x13g\x08g\x1f0\x00lz{\x97wíOá\n\x00-\x00 \x006\x00 \x00-' metadata={'source': 'Toyota_2023.pdf', 'page': 9}

page_content='連結財務諸 表\n１．連結財政状態計算 書\n（単位：百万円）\n前連結会計年 度\n（2022年３月31日）当連結会計年度\n（2023年３月31日）\n資産\n流動資産\n現金及び現金同等 物 6,113,655 7,516,966\n営業債権及びその他の債 権 3,142,832 3,586,130\n金融事業に係る債 権 7,181,327 8,279,806\nその他の金融資 産 2,507,248 1,715,675\n棚卸資産 3,821,356 4,255,614\n未収法人所得 税 163,925 218,704\nその他の流動資 産 791,947 886,885\n流動資産合 計 23,722,290 26,459,781\n非流動資 産\n持分法で会計処理されて いる投資 4,837,895 5,227,345\n金融事業に係る債 権 14,583,130 16,491,045\nその他の金融資 産 9,517,267 10,556,431\n有形固定資 産\n土地 1,361,791 1,426,370\n建物 5,284,620 5,464,811\n機械装置 13,982,362 14,796,619\n賃貸用車両及び器 具 6,781,229 6,774,427\n建設仮勘 定 565,528 846,866\n小計 27,975,530 29,309,093\n減価償却累計額及び減損 損失\n累計額＜控除＞△15,648,890 △16,675,119\n有形固定資産合 計 12,326,640 12,633,974\n使用権資 産 448,412 491,368\n無形資産 1,191,966 1,249,122\n繰延税金資 産 342,202 387,427\nその他の非流動資 産 718,968 806,687\n非流動資産合 計 43,966,482 47,843,399\n資産合計 67,688,771 74,303,180\n\u3000\n0È0è0¿\x81êRÕ\x8eÊ21\x00 \x00(\x007\x002\x000\x003\x00)\x00 \x002\x000\x002\x003^tÿ\x13g\x08g\x1f0\x00lz{\x97wíOá\n\x00-\x00 \x007\x00 \x00-' metadata={'source': 'Toyota_2023.pdf', 'page': 10}

page_content='（単位：百万円）\n前連結会計年 度\n（2022年３月31日）当連結会計年度\n（2023年３月31日）\n負債\n流動負債\n営業債務及びその他の債 務 4,292,092 4,986,309\n有利子負 債 11,187,839 12,305,639\n未払費用 1,520,446 1,552,345\nその他の金融負 債 1,046,050 1,392,397\n未払法人所得 税 826,815 404,606\n品質保証に係る負 債 1,555,711 1,686,357\nその他の流動負 債 1,413,208 1,632,063\n流動負債合 計 21,842,161 23,959,715\n非流動負 債\n有利子負 債 15,308,519 17,074,634\nその他の金融負 債 461,583 533,710\n退職給付に係る負 債 1,022,749 1,065,508\n繰延税金負 債 1,354,794 1,802,346\nその他の非流動負 債 544,145 603,052\n非流動負債合 計 18,691,790 21,079,251\n負債合計 40,533,951 45,038,967\n資本\n資本金 397,050 397,050\n資本剰余 金 498,575 498,728\n利益剰余 金 26,453,126 28,343,296\nその他の資本の構成要 素 2,203,254 2,836,195\n自己株式 △3,306,037 △3,736,562\n親会社の所有者に帰属す る持分合 計 26,245,969 28,338,706\n非支配持 分 908,851 925,507\n資本合計 27,154,820 29,264,213\n負債及び資本合 計 67,688,771 74,303,180\n\u3000\n0È0è0¿\x81êRÕ\x8eÊ21\x00 \x00(\x007\x002\x000\x003\x00)\x00 \x002\x000\x002\x003^tÿ\x13g\x08g\x1f0\x00lz{\x97wíOá\n\x00-\x00 \x008\x00 \x00-' metadata={'source': 'Toyota_2023.pdf', 'page': 11}

page_content='２．連結損益計算書及び連結 包括利益計算 書\n連結損益計算 書\n（単位：百万円）\n前連結会計年度\n（2022年３月31日に\n終了した１年間）当連結会計年度\n（2023年３月31日に\n終了した１年間）\n営業収益\n商品・製品売上収 益 29,073,428 34,367,619\n金融事業に係る金融収 益 2,306,079 2,786,679\n営業収益合 計 31,379,507 37,154,298\n売上原価並びに販売費及 び\n一般管理 費\n売上原価 24,250,784 29,128,561\n金融事業に係る金融費 用 1,157,050 1,712,721\n販売費及び一般管理 費 2,975,977 3,587,990\n売上原価並びに販売費及 び\n一般管理費合 計28,383,811 34,429,273\n営業利益 2,995,697 2,725,025\n持分法による投資損 益 560,346 643,063\nその他の金融収 益 334,760 379,350\nその他の金融費 用 △43,997 △125,113\n為替差損 益＜純額＞ 216,187 124,516\nその他＜純額＞ △72,461 △78,109\n税引前利 益 3,990,532 3,668,733\n法人所得税費 用 1,115,918 1,175,765\n当期利益 2,874,614 2,492,967\n当期利益の帰 属\n親会社の所有 者 2,850,110 2,451,318\n非支配持 分 24,504 41,650\n当期利益 2,874,614 2,492,967\n（単位：円）\n１株当たり親会社の所有者 に帰属す る\n当期利益\n基本的 205.23 179.47\n希薄化後 205.23 －\n\u3000\n0È0è0¿\x81êRÕ\x8eÊ21\x00 \x00(\x007\x002\x000\x003\x00)\x00 \x002\x000\x002\x003^tÿ\x13g\x08g\x1f0\x00lz{\x97wíOá\n\x00-\x00 \x009\x00 \x00-' metadata={'source': 'Toyota_2023.pdf', 'page': 12}

page_content='連結包括利益計算 書\n（単位：百万円）\n前連結会計年度\n（2022年３月31日に\n終了した１年間）当連結会計年度\n（2023年３月31日に\n終了した１年間）\n当期利益 2,874,614 2,492,967\nその他の包括利 益（税効果考慮 後）\n純損益に振り替えられる ことのない項 目\nその他の包括利益を通じ て公正価値 で\n測定する金融資産の公正 価値変動△49,242 99,223\n確定給付制度の再測 定 136,250 65,153\n持分法で会計処理されて いる投資 の\nその他の包括利益に対す る持分相当 額113,641 △77,148\n合計 200,648 87,228\n純損益に振り替えられる 可能性のあ る\n項目\n在外営業活動体の為替換 算差額 902,844 676,042\nその他の包括利益を通じ て公正価値 で\n測定する金融資産の公正 価値変動△154,174 △115,738\n持分法で会計処理されて いる投資 の\nその他の包括利益に対す る持分相当 額193,811 180,181\n合計 942,480 740,485\nその他の包括利 益（税効果考慮 後）合計 1,143,129 827,713\n当期包括利 益 4,017,742 3,320,681\n当期包括利益の帰 属\n親会社の所有 者 3,954,350 3,251,090\n非支配持 分 63,392 69,591\n当期包括利 益 4,017,742 3,320,681\n\u3000\n0È0è0¿\x81êRÕ\x8eÊ21\x00 \x00(\x007\x002\x000\x003\x00)\x00 \x002\x000\x002\x003^tÿ\x13g\x08g\x1f0\x00lz{\x97wíOá\n\x00-\x00 \x001\x000\x00 \x00-' metadata={'source': 'Toyota_2023.pdf', 'page': 13}

page_content='３．連結持分変動計算 書\n前連結会計年 度（2022年３月31日に終了し た１年間）\n（単位：百万円）\n親会社の所有者に帰属す る持分\n非支配持分 資本合計\n資本金 資本剰余金 利益剰余金その他の\n資本の\n構成要素自己株式 合計\n2021年４月１日現在残 高 397,050497,27524,104,1761,307,726△2,901,68023,404,547883,78224,288,329\n当期包括利 益\n当期利益 － －2,850,110 － －2,850,110 24,5042,874,614\nその他の包括利 益\n（税効果考慮 後）－ － －1,104,240 －1,104,240 38,8891,143,129\n当期包括利益合 計 － －2,850,1101,104,240 －3,954,350 63,3924,017,742\n所有者との取引 等\n配当金の支 払 － －△709,872 － －△709,872△51,723△761,595\n自己株式の取 得 － － － －△404,718△404,718 －△404,718\n自己株式の処 分 － 227 － － 362 588 － 588\nその他 － 1,074 － － － 1,074 13,400 14,473\n所有者との取引等合 計 － 1,300△709,872 －△404,357△1,112,928△38,323△1,151,252\nその他の資本の構成要 素\nから利益剰余金への振 替－ － 208,712△208,712 － － － －\n2022年３月31日現在残 高 397,050498,57526,453,1262,203,254△3,306,03726,245,969908,85127,154,820\n当連結会計年 度（2023年３月31日に終了し た１年間）\n（単位：百万円）\n親会社の所有者に帰属す る持分\n非支配持分 資本合計\n資本金 資本剰余金 利益剰余金その他の\n資本の\n構成要素自己株式 合計\n2022年４月１日現在残 高 397,050498,57526,453,1262,203,254△3,306,03726,245,969908,85127,154,820\n当期包括利 益\n当期利益 － －2,451,318 － －2,451,318 41,6502,492,967\nその他の包括利 益\n（税効果考慮 後）－ － － 799,772 － 799,772 27,941827,713\n当期包括利益合 計 － －2,451,318799,772 －3,251,090 69,5913,320,681\n所有者との取引 等\n配当金の支 払 － －△727,980 － －△727,980△84,986△812,966\n自己株式の取 得 － － － －△431,099△431,099 －△431,099\n自己株式の処 分 － 334 － － 573 907 － 907\nその他 － △181 － － － △181 32,052 31,871\n所有者との取引等合 計 － 152△727,980 －△430,526△1,158,353△52,934△1,211,287\nその他の資本の構成要 素\nから利益剰余金への振 替－ － 166,831△166,831 － － － －\n2023年３月31日現在残 高 397,050498,72828,343,2962,836,195△3,736,56228,338,706925,50729,264,213\n\u3000\n0È0è0¿\x81êRÕ\x8eÊ21\x00 \x00(\x007\x002\x000\x003\x00)\x00 \x002\x000\x002\x003^tÿ\x13g\x08g\x1f0\x00lz{\x97wíOá\n\x00-\x00 \x001\x001\x00 \x00-' metadata={'source': 'Toyota_2023.pdf', 'page': 14}

page_content='４．連結キャッシュ ・フロー計算 書\n（単位：百万円）\n前連結会計年度\n（2022年３月31日に\n終了した１年間）当連結会計年度\n（2023年３月31日に\n終了した１年間）\n営業活動によるキャッシ ュ・フロー\n当期利益 2,874,614 2,492,967\n減価償却費及び償却 費 1,821,880 2,039,904\n金融事業に係る利息収益 及び利息費 用 △354,102 △694,331\n持分法による投資損 益 △560,346 △643,063\n法人所得税費 用 1,115,918 1,175,765\n資産及び負債の増減ほ か △1,130,667 △1,502,482\n営業債権及びその他の債 権の増減（△は増加） 118,652 △532,432\n金融事業に係る債権の増 減（△は増加） △1,213,234 △1,760,288\n棚卸資産の増 減（△は増加） △725,285 △350,550\nその他の流動資産の増 減（△は増加） 71,314 △61,538\n営業債務及びその他の債 務の増減（△は減少） 152,399 712,400\nその他の流動負債の増 減（△は減少） 410,546 545,666\n退職給付に係る負債の増 減（△は減少） 60,419 21,213\nその他 △5,478 △76,953\n利息の受取 額 835,739 1,516,404\n配当金の受取 額 347,387 460,351\n利息の支払 額 △418,043 △593,216\n法人所得税の支払 額 △809,763 △1,297,224\n営業活動によるキャッシ ュ・フロー 3,722,615 2,955,076\n投資活動によるキャッシ ュ・フロー\n有形固定資産の購 入＜賃貸資産を除 く＞ △1,197,266 △1,450,196\n賃貸資産の購 入 △2,286,893 △1,907,356\n有形固定資産の売 却＜賃貸資産を除 く＞ 37,749 56,436\n賃貸資産の売 却 1,542,132 1,659,161\n無形資産の取 得 △346,085 △348,280\n公社債及び株式の購 入 △2,427,911 △1,150,214\n公社債及び株式の売 却 282,521 393,982\n公社債の満期償 還 1,920,116 939,747\nその他 1,898,143 207,829\n投資活動によるキャッシ ュ・フロー △577,496 △1,598,890\n財務活動によるキャッシ ュ・フロー\n短期有利子負債の純増減 額（△は減少） △579,216 239,689\n長期有利子負債の増 加 8,122,678 9,276,918\n長期有利子負債の返 済 △8,843,665 △8,353,033\n親会社の所有者への配当 金の支払 額 △709,872 △727,980\n非支配持分への配当金の 支払額 △51,723 △84,986\n自己株式の取 得（△）及び処分 △404,718 △431,099\nその他 － 24,310\n財務活動によるキャッシ ュ・フロー △2,466,516 △56,180\n現金及び現金同等物に対 する為替変動の影響 額 334,195 103,305\n現金及び現金同等物純増 減額（△は減少） 1,012,798 1,403,311\n現金及び現金同等物期首 残高 5,100,857 6,113,655\n現金及び現金同等物期末 残高 6,113,655 7,516,966\n５．継続企業の前提に関する 注記\n該当事項はありませ ん。\n\u3000\n0È0è0¿\x81êRÕ\x8eÊ21\x00 \x00(\x007\x002\x000\x003\x00)\x00 \x002\x000\x002\x003^tÿ\x13g\x08g\x1f0\x00lz{\x97wíOá\n\x00-\x00 \x001\x002\x00 \x00-' metadata={'source': 'Toyota_2023.pdf', 'page': 15}

page_content='６．セグメント情報\n（１）報告セグメントに関する 情報\n前連結会計年 度（2022年３月31日現在あるいは同 日に終了し た１年間）\n（単位：百万円）\n自動車 金融 その他消去又は\n全社連結\n営業収益\n外部顧客への営業収 益 28,531,993 2,306,079 541,436 － 31,379,507\nセグメント間 の\n営業収益73,745 17,947 588,441 △680,133 －\n計 28,605,738 2,324,026 1,129,876 △680,13331,379,507\n営業費用 26,321,448 1,667,025 1,087,575 △692,23728,383,811\n営業利益 2,284,290 657,001 42,302 12,104 2,995,697\n資産合計 24,341,73731,681,472 3,091,011 8,574,55167,688,771\n持分法で会計処 理\nされている投 資4,354,085 79,414 258,750 145,646 4,837,895\n減価償却費及び償却 費 1,026,834 761,801 33,245 － 1,821,880\n資本的支 出 1,422,429 2,156,339 51,200 △18,381 3,611,587\n当連結会計年 度（2023年３月31日現在あるいは同 日に終了し た１年間）\n（単位：百万円）\n自動車 金融 その他消去又は\n全社連結\n営業収益\n外部顧客への営業収 益 33,776,870 2,786,679 590,749 － 37,154,298\nセグメント間 の\n営業収益43,131 22,968 634,194 △700,293 －\n計 33,820,000 2,809,647 1,224,943 △700,29337,154,298\n営業費用 31,639,363 2,372,131 1,121,492 △703,71334,429,273\n営業利益 2,180,637 437,516 103,451 3,420 2,725,025\n資産合計 26,321,85835,525,441 2,946,994 9,508,88774,303,180\n持分法で会計処 理\nされている投 資4,717,231 92,903 272,752 144,460 5,227,345\n減価償却費及び償却 費 1,205,687 799,156 35,062 － 2,039,904\n資本的支 出 1,688,114 1,786,373 38,748 △17,015 3,496,219\n（注）全社資産は主 に、全社共通の目的で保有し ている現金及び現金同等 物ならびにその他の包括 利益を通じて公 正\n価値で測定する金融資産 で構成されてお り、2022年３月31日および2023 年３月31日現在の残高 は、それぞれ\n10,020,460百万円および11,10 1,175百万円で す。\n\u3000\n0È0è0¿\x81êRÕ\x8eÊ21\x00 \x00(\x007\x002\x000\x003\x00)\x00 \x002\x000\x002\x003^tÿ\x13g\x08g\x1f0\x00lz{\x97wíOá\n\x00-\x00 \x001\x003\x00 \x00-' metadata={'source': 'Toyota_2023.pdf', 'page': 16}

page_content='（２）自動車等セグメントと金 融セグメントを区分した 連結財務諸 表\n自動車等セグメントと金 融セグメントを区分した 連結財政状態計算 書\n（単位：百万円）\n前連結会計年 度\n（2022年３月31日）当連結会計年 度\n（2023年３月31日）\n資産\n（自動車等）\n流動資産\n現金及び現金同等 物 4,299,522 5,548,398\n営業債権及びその他の債 権 3,184,782 3,594,057\nその他の金融資 産 2,028,649 849,779\n棚卸資産 3,821,356 4,255,614\nその他の流動資 産 746,134 749,078\n流動資産合 計 14,080,444 14,996,926\n非流動資 産\n有形固定資 産 7,302,017 7,729,000\nその他の非流動資 産 15,769,015 17,337,727\n非流動資産合 計 23,071,032 25,066,727\n資産合計 37,151,476 40,063,653\n（金融）\n流動資産\n現金及び現金同等 物 1,814,133 1,968,568\n営業債権及びその他の債 権 206,588 286,960\n金融事業に係る債 権 7,181,327 8,279,806\nその他の金融資 産 1,058,620 1,680,242\nその他の流動資 産 221,738 362,660\n流動資産合 計 10,482,407 12,578,237\n非流動資 産\n金融事業に係る債 権 14,583,130 16,491,045\n有形固定資 産 5,024,625 4,904,975\nその他の非流動資 産 1,591,311 1,551,183\n非流動資産合 計 21,199,065 22,947,204\n資産合計 31,681,472 35,525,441\n（消去）\n資産消去 計 △1,144,177 △1,285,914\n（連結）\n資産合計 67,688,771 74,303,180\n（注）自動車等セグメントは全 社資産を含んでいま す。\u3000\n\u3000\n0È0è0¿\x81êRÕ\x8eÊ21\x00 \x00(\x007\x002\x000\x003\x00)\x00 \x002\x000\x002\x003^tÿ\x13g\x08g\x1f0\x00lz{\x97wíOá\n\x00-\x00 \x001\x004\x00 \x00-' metadata={'source': 'Toyota_2023.pdf', 'page': 17}

page_content='（単位：百万円）\n前連結会計年 度\n（2022年３月31日）当連結会計年 度\n（2023年３月31日）\n負債\n（自動車等）\n流動負債\n営業債務及びその他の債 務 4,023,857 4,689,034\n有利子負 債 1,041,557 1,170,114\n未払費用 1,421,194 1,446,697\n未払法人所得 税 695,888 361,000\nその他の流動負 債 2,778,172 3,266,095\n流動負債合 計 9,960,668 10,932,939\n非流動負 債\n有利子負 債 1,538,884 1,553,622\n退職給付に係る負 債 1,004,558 1,047,430\nその他の非流動負 債 1,830,146 1,867,028\n非流動負債合 計 4,373,588 4,468,080\n負債合計 14,334,256 15,401,019\n（金融）\n流動負債\n営業債務及びその他の債 務 477,550 547,511\n有利子負 債 10,576,910 11,583,602\n未払費用 124,088 128,994\n未払法人所得 税 130,927 43,607\nその他の流動負 債 1,414,606 1,841,562\n流動負債合 計 12,724,080 14,145,275\n非流動負 債\n有利子負 債 13,882,650 15,627,943\n退職給付に係る負 債 18,190 18,078\nその他の非流動負 債 722,257 1,135,862\n非流動負債合 計 14,623,097 16,781,883\n負債合計 27,347,177 30,927,158\n（消去）負債消去 計 △1,147,482 △1,289,211\n（連結）負債合計 40,533,951 45,038,967\n資本\n（連結）親会社の所有者に帰属す る持分合 計 26,245,969 28,338,706\n（連結）非支配持 分 908,851 925,507\n（連結）資本合計 27,154,820 29,264,213\n（連結）負債及び資本合 計 67,688,771 74,303,180\n\u3000\n0È0è0¿\x81êRÕ\x8eÊ21\x00 \x00(\x007\x002\x000\x003\x00)\x00 \x002\x000\x002\x003^tÿ\x13g\x08g\x1f0\x00lz{\x97wíOá\n\x00-\x00 \x001\x005\x00 \x00-' metadata={'source': 'Toyota_2023.pdf', 'page': 18}

page_content='自動車等セグメントと金 融セグメントを区分した 連結損益計算 書\n（単位：百万円）\n前連結会計年 度\n（2022年３月31日に\n\u3000終了した１年間）当連結会計年 度\n（2023年３月31日に\n\u3000終了した１年間）\n（自動車等）\n営業収益 29,104,564 34,409,011\n売上原価 24,250,860 29,132,715\n販売費及び一般管理 費 2,518,182 2,990,316\n営業利益 2,335,522 2,285,980\nその他の収益 ・費用（△）＜純額＞ 998,001 943,777\n税引前利 益 3,333,522 3,229,757\n法人所得税費 用 944,594 1,040,864\n当期利益 2,388,928 2,188,893\n当期利益の帰 属\n親会社の所有 者 2,369,399 2,152,509\n非支配持 分 19,529 36,384\n（金融）\n営業収益 2,324,026 2,809,647\n売上原価 1,178,509 1,741,117\n販売費及び一般管理 費 488,517 631,014\n営業利益 657,001 437,516\nその他の収益 ・費用（△）＜純額＞ 16 △5,013\n税引前利 益 657,017 432,503\n法人所得税費 用 171,327 134,903\n当期利益 485,690 297,600\n当期利益の帰 属\n親会社の所有 者 480,716 292,334\n非支配持 分 4,974 5,266\n（消去）\n当期利益消 去 △4 6,475\n（連結）\n当期利益 2,874,614 2,492,967\n当期利益の帰 属\n親会社の所有 者 2,850,110 2,451,318\n非支配持 分 24,504 41,650\n\u3000\n0È0è0¿\x81êRÕ\x8eÊ21\x00 \x00(\x007\x002\x000\x003\x00)\x00 \x002\x000\x002\x003^tÿ\x13g\x08g\x1f0\x00lz{\x97wíOá\n\x00-\x00 \x001\x006\x00 \x00-' metadata={'source': 'Toyota_2023.pdf', 'page': 19}

page_content='自動車等セグメントと金 融セグメントを区分した 連結キャッシュ ・フロー計算 書\n（単位：百万円）\n前連結会計年 度\n（2022年３月31日に\n終了した１年間）当連結会計年 度\n（2023年３月31日に\n終了した１年間）\n（自動車等）\n営業活動によるキャッシ ュ・フロー\n当期利益 2,388,928 2,188,893\n減価償却費及び償却 費 1,060,079 1,240,749\n持分法による投資損 益 △552,515 △633,324\n法人所得税費 用 944,594 1,040,864\n資産及び負債の増減ほ か △572,082 463,871\n利息の受取 額 100,118 234,945\n配当金の受取 額 342,646 454,752\n利息の支払 額 △40,780 △28,206\n法人所得税の支払 額 △544,887 △1,280,341\n営業活動によるキャッシ ュ・フロー 3,126,101 3,682,203\n投資活動によるキャッシ ュ・フロー\n有形固定資産の購 入＜賃貸資産を除 く＞ △1,186,900 △1,439,724\n賃貸資産の購 入 △151,456 △147,792\n有形固定資産の売 却＜賃貸資産を除 く＞ 36,219 54,572\n賃貸資産の売 却 45,183 44,195\n無形資産の取 得 △335,436 △333,295\n公社債及び株式の購 入 △1,904,588 △503,977\n公社債及び株式の売却及 び公社債の満期償 還 1,989,345 892,814\nその他 1,856,069 236,351\n投資活動によるキャッシ ュ・フロー 348,436 △1,196,856\n財務活動によるキャッシ ュ・フロー\n短期有利子負債の純増減 額（△は減少） △164,899 142,688\n長期有利子負債の増 加 513,371 474,535\n長期有利子負債の返 済 △1,818,653 △637,982\n親会社の所有者への配当 金の支払 額 △709,872 △727,980\n非支配持分への配当金の 支払額 △49,629 △79,782\n自己株式の取 得（△）及び処分 △404,718 △431,099\nその他 － 21,458\n財務活動によるキャッシ ュ・フロー △2,634,401 △1,238,161\n現金及び現金同等物に対 する為替変動の影響 額 185,237 1,690\n現金及び現金同等物純増 減額（△は減少） 1,025,373 1,248,876\n現金及び現金同等物期首 残高 3,274,149 4,299,522\n現金及び現金同等物期末 残高 4,299,522 5,548,398\n\u3000\n\u3000\n0È0è0¿\x81êRÕ\x8eÊ21\x00 \x00(\x007\x002\x000\x003\x00)\x00 \x002\x000\x002\x003^tÿ\x13g\x08g\x1f0\x00lz{\x97wíOá\n\x00-\x00 \x001\x007\x00 \x00-' metadata={'source': 'Toyota_2023.pdf', 'page': 20}

page_content='（単位：百万円）\n前連結会計年 度\n（2022年３月31日に\n終了した１年間）当連結会計年 度\n（2023年３月31日に\n終了した１年間）\n（金融）\n営業活動によるキャッシ ュ・フロー\n当期利益 485,690 297,600\n減価償却費及び償却 費 761,801 799,156\n金融事業に係る利息収益 及び利息費 用 △360,837 △703,971\n持分法による投資損 益 △7,831 △9,739\n法人所得税費 用 171,327 134,903\n資産及び負債の増減ほ か △623,051 △1,958,779\n利息の受取 額 742,364 1,291,100\n配当金の受取 額 4,740 5,599\n利息の支払 額 △384,006 △574,650\n法人所得税の支払 額 △264,876 △16,883\n営業活動によるキャッシ ュ・フロー 525,321 △735,664\n投資活動によるキャッシ ュ・フロー\n有形固定資産の購 入＜賃貸資産を除 く＞ △10,366 △10,472\n賃貸資産の購 入 △2,135,437 △1,759,564\n有形固定資産の売 却＜賃貸資産を除 く＞ 1,530 1,865\n賃貸資産の売 却 1,496,949 1,614,965\n無形資産の取 得 △10,650 △14,985\n公社債及び株式の購 入 △523,323 △646,237\n公社債及び株式の売却及 び公社債の満期償 還 213,291 440,915\nその他 113,635 △30,385\n投資活動によるキャッシ ュ・フロー △854,370 △403,898\n財務活動によるキャッシ ュ・フロー\n短期有利子負債の純増減 額（△は減少） △488,495 171,293\n長期有利子負債の増 加 7,800,854 8,892,261\n長期有利子負債の返 済 △7,142,750 △7,868,820\n非支配持分への配当金の 支払額 △2,094 △5,204\nその他 － 2,853\n財務活動によるキャッシ ュ・フロー 167,516 1,192,382\n現金及び現金同等物に対 する為替変動の影響 額 148,958 101,615\n現金及び現金同等物純増 減額（△は減少） △12,575 154,436\n現金及び現金同等物期首 残高 1,826,707 1,814,133\n現金及び現金同等物期末 残高 1,814,133 1,968,568\n（連結）\n現金及び現金同等物に対 する為替変動の影響 額 334,195 103,305\n現金及び現金同等物純増 減額（△は減少） 1,012,798 1,403,311\n現金及び現金同等物期首 残高 5,100,857 6,113,655\n現金及び現金同等物期末 残高 6,113,655 7,516,966\n\u3000\n0È0è0¿\x81êRÕ\x8eÊ21\x00 \x00(\x007\x002\x000\x003\x00)\x00 \x002\x000\x002\x003^tÿ\x13g\x08g\x1f0\x00lz{\x97wíOá\n\x00-\x00 \x001\x008\x00 \x00-' metadata={'source': 'Toyota_2023.pdf', 'page': 21}

page_content='（３）地域に関する情 報\n前連結会計年 度（2022年３月31日現在あるいは同 日に終了し た１年間）\n（単位：百万円）\n日本 北米 欧州 アジア その他消去又は\n全社連結\n営業収益\n外部顧客へ の\n営業収益8,214,74010,897,9463,692,2145,778,1152,796,493 －31,379,507\n所在地間 の\n営業収益7,776,696268,534175,633752,452131,690△9,105,004 －\n計 15,991,43611,166,4793,867,8476,530,5662,928,183△9,105,00431,379,507\n営業費用 14,567,99110,600,6953,704,8745,858,2162,690,014△9,037,98028,383,811\n営業利益 1,423,445565,784162,973672,350238,169△67,0242,995,697\n資産合計 21,502,15523,353,8125,711,2717,461,8124,309,2485,350,47467,688,771\n非流動資 産 5,501,0466,251,499891,146977,235537,631 －14,158,559\n当連結会計年 度（2023年３月31日現在あるいは同 日に終了し た１年間）\n（単位：百万円）\n日本 北米 欧州 アジア その他消去又は\n全社連結\n営業収益\n外部顧客へ の\n営業収益9,122,28213,509,0274,097,5377,076,9223,348,530 －37,154,298\n所在地間 の\n営業収益8,460,914334,874176,198967,984123,663△10,063,633 －\n計 17,583,19613,843,9014,273,7358,044,9063,472,193△10,063,63337,154,298\n営業費用 15,681,73313,918,6374,216,2767,330,4553,240,832△9,958,65934,429,273\n営業利益 ・損失（△）1,901,463△74,73657,460714,451231,362△104,9742,725,025\n資産合計 23,241,33426,024,7346,813,4747,908,5204,726,3735,588,74574,303,180\n非流動資 産 5,658,8596,255,5611,042,7261,031,057565,377 －14,553,580\n（注）１全社資産は主 に、全社共通の目的で保有し ている現金及び現金同等 物ならびにその他の包括 利益を通じて公\n正価値で測定する金融資 産で構成されてお り、2022年３月31日および2023 年３月31日現在の残高 は、それぞ\nれ10,020,460 百万円および11,10 1,175百万円で す。\n２「その他」は、中南米、オセアニ ア、アフリカ、中東からなりま す。\n３非流動資産は金融商 品、繰延税金資 産、確定給付資産の純額およ び保険契約から生じる権 利を含んでいませ\nん。\n\u3000\n0È0è0¿\x81êRÕ\x8eÊ21\x00 \x00(\x007\x002\x000\x003\x00)\x00 \x002\x000\x002\x003^tÿ\x13g\x08g\x1f0\x00lz{\x97wíOá\n\x00-\x00 \x001\x009\x00 \x00-' metadata={'source': 'Toyota_2023.pdf', 'page': 22}

page_content='７．１株当たり情 報\n基本的および希薄化後１株当たり親会社の所有者に帰属する当期利益の差異の調整は次のとお\nりです。\n金額：百万円 単 位：千株 金 額：円\n親会社の所有者 に\n帰属する\n当期利益加重平均\n普通株式数１株当たり親会社 の\n所有者に帰属す る\n当期利益\n2022年３月31日に終了し た１年間：\n親会社の所有者に帰属す る当期利 益 2,850,110\n基本的１株当たり\n親会社の所有者に帰属す る当期利 益2,850,110 1 3,887,348 205.23\n希薄化の影 響\n希薄化効果を有する種類 株式 23 311\n希薄化後１株当たり\n親会社の所有者に帰属す る当期利 益2,850,132 1 3,887,659 205.23\n2023年３月31日に終了し た１年間：\n親会社の所有者に帰属す る当期利 益 2,451,318\n基本的１株当たり\n親会社の所有者に帰属す る当期利 益2,451,318 1 3,658,382 179.47\n希薄化の影 響\n希薄化効果を有する種類 株式 － －\n希薄化後１株当たり\n親会社の所有者に帰属す る当期利 益－ － －\nトヨタは 、IFRSで要求される開示に加え 、日本における財務諸表利用者に有用な情報を提供す\nるため、以下の情報を開示してい ます。\n１株当たり親会社の所有者に帰属する持分は次のとおりです。なお 、１株当たり親会社の所有\n者に帰属する持分は 、連結財政状態計算書の親会社の所有者に帰属する持分を期末発行済普通株\n式数（自己株式を除 く）で除すことにより計算し ています。\n金額：百万円 単 位：千株 金 額：円\n親会社の所有者 に\n帰属する持分期末発行 済\n普通株式 数（自己\n株式を除 く）１株当たり\n親会社の所有者 に\n帰属する持分\n2022年３月31日 26,245,969 1 3,778,302 1,904.88\n2023年３月31日 28,338,706 1 3,565,180 2,089.08\n（注） １ 2 021年10月１日付で普通株式１株につき５株の割合で株式分割を行っています。前連結会計年度の期\n首に当該株式分割が行われたと仮定して 、基本的１株当たり親会社の所有者に帰属する当期利益 、希\n薄化後１株当たり親会社の所有者に帰属する当期利益および1株当たり親会社の所有者に帰属する持分\nを算定していま す。\n２ 2021年４月２日に第１回ＡＡ型種類株式の残存する全部の取得および2 021年４月３日に第１回ＡＡ型\n種類株式の全部の消却を完了しており 、2023年３月3 1日に終了した１年間において 、潜在株式が存在\nしないため希薄化 後１株当たり親会社の所有者 に帰属する当期利益は記 載していませ ん。\n0È0è0¿\x81êRÕ\x8eÊ21\x00 \x00(\x007\x002\x000\x003\x00)\x00 \x002\x000\x002\x003^tÿ\x13g\x08g\x1f0\x00lz{\x97wíOá\n\x00- \x002\x000\x00 \x00-' metadata={'source': 'Toyota_2023.pdf', 'page': 23}

page_content='８．重要な後発事 象\n\u3000該当事項はありませ ん。\n\u3000\n0È0è0¿\x81êRÕ\x8eÊ21\x00 \x00(\x007\x002\x000\x003\x00)\x00 \x002\x000\x002\x003^tÿ\x13g\x08g\x1f0\x00lz{\x97wíOá\n\x00-\x00 \x002\x001\x00 \x00-' metadata={'source': 'Toyota_2023.pdf', 'page': 24}

page_content="補足資料（連結決算）\n＜IFRS＞\n2022/3月期 2023/3月期 2024/3月期\n第1四半期 第2四半期 第3四半期 第4四半期 第1四半期 第2四半期 第3四半期 第4四半期 通期\n2,180 1,756 2,077 2,145 8,158 1,936 2,179 2,243 2,336 8,694\n1,025 826 945 942 3,738 802 934 975 1,078 3,789\n[ 262 ][ 194 ][ 257 ][ 264 ][ 977 ][ 208 ][ 240 ][ 285 ][ 269 ][ 1,002 ]\n1,155 930 1,132 1,203 4,420 1,135 1,245 1,268 1,258 4,905\n[ 110 ][ 121 ][ 140 ][ 145 ][ 515 ][ 121 ][ 157 ][ 164 ][ 156 ][ 597 ]\n519 395 403 435 1,752 452 460 415 441 1,768\n199 110 201 197 707 201 183 210 176 771\n324 316 419 440 1,499 372 468 510 508 1,859\n83 84 81 87 335 104 105 95 95 398\n29 25 29 45 127 5 28 38 38 109\n2,148 1,946 2,003 2,134 8,230 2,013 2,146 2,331 2,331 8,822 9,600\n500 445 482 498 1,924 403 466 532 668 2,069 2,320\n[ 146 ][ 128 ][ 133 ][ 155 ][ 561 ][ 126 ][ 144 ][ 169 ][ 182 ][ 621 ] [ 620 ]\n1,648 1,501 1,521 1,637 6,306 1,610 1,680 1,799 1,664 6,753 7,280\n[ 63 ][ 63 ][ 78 ][ 77 ][ 280 ][ 68 ][ 78 ][ 92 ][ 83 ][ 321 ] [ 280 ]\n661 621 522 589 2,394 635 610 607 555 2,407 2,660\n281 229 250 257 1,017 248 231 277 273 1,030 1,130\n365 341 413 425 1,543 361 456 476 458 1,751 1,870\n105 105 111 111 432 123 128 144 115 509 500\n74 74 60 75 282 76 67 68 54 265 290\n51 45 58 65 219 36 61 68 61 226 270\n108 84 106 114 412 130 126 157 145 559 560\n2 2 2 2 7 2 2 2 2 7\n2,759 2,506 2,522 2,594 10,381 2,544 2,625 2,721 2,669 10,558 11,380(2021/7-9月)\n（国内）(2021/4-6月)\n欧州北米 車 両 生 産 台 数  (千台)\n（海外）[うち\u3000ダイハツ・日野]\n[うち\u3000ダイハツ・日野]\nアジア\nその他中南米\n 車 両 販 売 台 数  (千台)\n 車両小売販売台数  (千台)\n\u3000\u3000[トヨタ・ダイハツ・日野]欧州通期\n（国内）\n[うち\u3000ダイハツ・日野]アフリカアジア\n[うち\u3000ダイハツ・日野]\n中南米\nアフリカ（海外）\n北米\n中東オセアニア見通し\n通期\n('23/4-'24/3) (2022/1-3月) (2021/10-12月) ('22/4-'23/3) (2022/10-12月) (2022/7-9月) (2023/1-3月) (2022/4-6月) ('21/4-'22/3)\n－補足1－" metadata={'source': 'Toyota_2023.pdf', 'page': 25}

page_content="補足資料（連結決算）\n＜IFRS＞\n2022/3月期 2023/3月期 2024/3月期\n第1四半期 第2四半期 第3四半期 第4四半期 第1四半期 第2四半期 第3四半期 第4四半期 通期\n(2021/7-9月) (2021/4-6月)通期見通し\n通期\n('23/4-'24/3) (2022/1-3月) (2021/10-12月) ('22/4-'23/3) (2022/10-12月) (2022/7-9月) (2023/1-3月) (2022/4-6月) ('21/4-'22/3)\n110 110 114 116 112 130 138 141 132 135\n132 130 130 130 131 138 139 144 142 141\n(注1)\n79,355 75,457 77,857 81,124 313,795 84,911 92,182 97,546 96,902 371,542\n40,966 36,955 40,961 41,029 159,914 38,997 42,901 45,822 48,110 175,831\n28,750 28,044 25,937 28,931 111,664 33,499 35,236 35,889 33,814 138,439\n9,869 9,002 10,002 9,803 38,678 9,730 9,393 11,556 12,056 42,737\n15,319 14,113 17,396 18,475 65,305 17,956 21,533 21,839 19,119 80,449\n6,882 7,043 7,022 8,333 29,281 8,287 9,075 9,071 8,287 34,721\n-22,433 -19,703 -23,463 -25,449 -91,050 -23,559 -25,958 -26,632 -24,485 -100,636\n72,412 68,838 70,738 74,067 286,057 77,201 84,089 88,714 88,194 338,200\n5,788 5,676 5,883 5,892 23,240 6,521 7,042 7,417 7,115 28,096\n2,412 2,543 2,669 3,673 11,298 2,623 2,710 3,122 3,792 12,249\n-1,257 -1,601 -1,434 -2,508 -6,801 -1,434 -1,660 -1,706 -2,200 -7,002\n9,974 7,499 7,843 4,638 29,956 5,786 5,627 9,566 6,269 27,250 30,000\n （営業利益率）   （％） ( 12.6 )( 9.9 )( 10.1 )( 5.7 )( 9.5 )( 6.8 )( 6.1 )( 9.8 )( 6.5 )( 7.3 ) ( 7.9 )\n5,185 2,905 3,965 2,178 14,234 3,367 5,211 6,834 3,601 19,014\n2,488 1,744 1,292 131 5,657 145 -856 -179 142 -747\n300 236 827 265 1,629 196 -655 626 407 574\n1,666 1,577 1,752 1,727 6,723 2,127 1,789 1,687 1,540 7,144\n653 827 517 382 2,381 458 640 661 553 2,313\n-319 207 -511 -47 -670 -508 -501 -63 23 -1,049\n7,800 5,776 5,953 3,311 22,842 4,492 4,537 8,188 4,588 21,806\n1,960 1,683 1,906 1,018 6,570 1,145 844 1,250 1,135 4,375\n216 -1 -11 219 423 183 203 184 462 1,034\n-3 40 -5 89 121 -34 43 -57 83 34\n1,614 995 1,544 1,448 5,603 1,710 1,335 1,700 1,683 6,430 6,900\n12,572 8,868 10,935 7,529 39,905 10,217 8,125 10,349 7,994 36,687 36,900\n( 15.8 )( 11.8 )( 14.0 )( 9.3 )( 12.7 )( 12.0 )( 8.8 )( 10.6 )( 8.3 )( 9.9 ) ( 9.7 )\n8,978 6,266 7,917 5,338 28,501 7,368 4,342 7,279 5,522 24,513 25,800\n （当期利益率）   （％） ( 11.3 )( 8.3 )( 10.2 )( 6.6 )( 9.1 )( 8.7 )( 4.7 )( 7.5 )( 5.7 )( 6.6 ) ( 6.8 )\n 配 当 総 額  （億円） ― 3,324 ― 3,857 7,182 ― 3,421 ― 4,747 8,169 (注2)\n 1 株当たり  （円） ― 120 ― 28 ― ― 25 ― 35 60(注3)\n 配 当 性 向\u3000（％） ― 21.9 ― 29.2 25.3 ― 29.2 ― 37.1 33.4\n215 2,284 664 881 4,046 1,213 742 1,410 945 4,310 (注4)\n― 1,499 ― 2,856 4,356 ― 1,499 ― 1,500 2,999(注4)\n(注5)北米\n自動車\n 持分法による投資損益 （億円）日本 営 業 収 益  （億円）\nその他\nその他自動車\n事\n\u3000\n業\n\u3000\n別所\n\u3000\n在\n\u3000\n地\n\u3000\n別欧州\nアジア\n欧州金融\n消去又は全社消去又は全社\n消去又は全社円／米ドル\nレート 円／ユーロ\n16,314,987 16,314,987371,365\n16,314,987 従 業 員 数  （人）\n3,262,997 自己株式取得額  （億円）\n  ＜還元ベース＞配\n当\n 自己株式取得額  （億円）\n  ＜実施ベース＞事\n\u3000\n業\n\u3000\n別\n 税 引 前 利 益 （億円）金融\n （税引前利益率） （％）\n 発行済株式数  （千株）\u3000 営 業 利 益  （億円）   \n所\n\u3000\n在\n\u3000\n地\n\u3000\n別372,817\n 親会社の所有者に帰属する当期利益  （億円）その他消去又は全社アジア日本\nその他北米372,286\n3,262,997為替\n372,434125(前提 )\n135(前提 )\n380,000372,817\n16,314,987375,235 376,971 375,396\n16,314,987377,369 375,235\n16,314,987 16,314,987 16,314,987\n－補足2－" metadata={'source': 'Toyota_2023.pdf', 'page': 26}

page_content="補足資料（連結決算）\n＜IFRS＞\n2022/3月期 2023/3月期 2024/3月期\n第1四半期 第2四半期 第3四半期 第4四半期 第1四半期 第2四半期 第3四半期 第4四半期 通期\n(2021/7-9月) (2021/4-6月)通期見通し\n通期\n('23/4-'24/3) (2022/1-3月) (2021/10-12月) ('22/4-'23/3) (2022/10-12月) (2022/7-9月) (2023/1-3月) (2022/4-6月) ('21/4-'22/3)\n2,741 2,760 2,948 2,792 11,242 3,022 3,184 2,988 3,220 12,416 12,400 (注6)\n2,417 2,292 2,617 2,745 10,072 2,843 2,897 2,981 3,127 11,850 12,500 (注7)\n1,168 992 1,254 1,214 4,630 1,226 1,263 1,180 1,458 5,128\n639 681 735 810 2,867 874 953 944 894 3,666\n183 193 176 208 762 221 204 209 223 859\n332 325 342 382 1,382 399 425 562 452 1,840\n93 100 107 129 430 122 50 84 97 355\n2,790 3,589 2,796 4,254 13,430 2,746 4,074 3,588 5,649 16,058 18,600 (注7)\n1,357 1,578 1,492 2,318 6,746 1,286 1,516 1,481 2,754 7,038\n745 1,519 722 1,048 4,036 979 1,753 1,201 1,627 5,561\n252 197 178 190 818 103 103 169 241 617\n369 244 361 512 1,487 257 655 634 681 2,229\n65 49 42 184 341 119 45 102 344 612\n103,186 102,708 101,933 105,173 105,173 105,322 110,046 102,787 113,137 113,137 (注8)\n616,511\n241,230\n15.1 10.3 12.8 8.3 11.5 11.0 6.3 10.5 7.9 9.0\n5.8 4.1 5.1 3.3 4.4 4.2 2.4 4.0 3.0 3.5\n559 569\n169 168\n第4四半期 通期\n(2023/1-3月) ('22/4-'23/3)\n 営業面の努力 3,800 6,800\n 為替変動の影響\u3000\u3000\u3000\u3000 \u3000\u3000\u3000 \u3000\u3000\u3000\u3000 \u3000 \u3000 2,350 12,800\n 原価改善の努力 -3,500 -12,900\n 設計面の改善 -3,700 -13,400\n 工場・物流部門の改善 200 500\n 諸経費の増減・低減努力 -1,100 -5,250\n その他 81 -4,156\n（営業利益 増減） 1,630 -2,706\n 営業外の影響 -1,165 -511\n 持分法による投資損益 235 827\n 法人所得税費用・非支配持分に帰属する当期利益 -280 -769\n\u3000（親会社の所有者に帰属する当期利益 増減）\u3000 184 -3,987\n（注1）\u3000期末時点の就業人員数（当社および連結子会社（以下、トヨタという）からトヨタ外への出向者を除き、トヨタ外からトヨタへの出向者を含む）を表示\n（注2）\u3000第２四半期欄の配当は中間配当、第４四半期欄の配当は期末配当、通期欄の配当は年間配当\n（注3）\u30002022年３月期の第２四半期の配当金の額は、株式分割前の実績の配当金の額を記載しています。\n（注3）\u30002022年３月期の年間配当金については、株式分割の実施により単純合算ができないため、表示していません。\n（注3）\u3000なお、株式分割を考慮しない場合の年間配当金は260円、株式分割を考慮する場合の年間配当金は52円です。\n（注4）\u3000単元未満株式の買取請求に基づく取得および取得に係る委託手数料を除く\n（注5）\u3000当該期の利益に対する株主還元（取得期間中は取得価額の上限、取得期間終了後は取得した実績を記載）\n（注6）\u3000研究開発費は報告期間中に発生した研究開発活動に係る支出であり、連結損益計算書上の研究開発費と異なる\n（注7）\u3000減価償却費・設備投資額はリース用車両および使用権資産を除く\n（注8）\u3000金融事業を除いた、現金および現金同等物、定期預金、公社債および信託ファンドへの投資北米\n欧州\nアジア所\n\u3000\n在\n\u3000\n地\n\u3000\n別日本\nアジア 減 価 償 却 費  （億円）\n 親会社の所有者に帰属する当期利益\n 増減要因（億円・概算） 連結子会社数  （社）\n （ストラクチャード・エンティティを含む） 資産合計当期利益率 （％）\n 持分法適用会社数  （社）\n （ジョイント・ベンチャーを含む） 親会社の所有者に帰属する持分  （億円）\n 親会社所有者帰属持分当期利益率 （％） 設 備 投 資 額  （億円）\nその他\n 総  資  産  （億円） 総 資 金 量  （億円）所\n\u3000\n在\n\u3000\n地\n\u3000\n別日本\n246,104 262,459676,887\n262,459617,588 636,180\n249,395北米\n欧州\n676,887その他 研 究 開 発 費  （億円）\n743,031\n283,387719,351\n273,748743,031\n283,387715,805\n275,358744,840\n280,372\n本決算要旨 には、当社（連結子会社を含む）の見通し等の将来に関する記述が含まれて います。\nこれらの将来に関する記述は、当社が現在入手している情報を基礎とした判断および仮定に基づいており、 判断や仮定に内在する不確定性および今後の事業運営や\n内外の状況変化等による変動可能性に照らし、将来における当社の実際の業績と大きく異なる可能性があります。\n上記の不確定性および変動可能性を有する要素は多数あり、以下のようなものが含まれます。\n・日本、北米、欧州、アジアおよび当社が営業活動を行っているその他の国の自動車市場に 影響を与える経済情勢、市場の需要ならびにそれらにおける競争環境\n・為替相場（主として日本円、米ドル、ユーロ、豪ドル、加ドルおよび英国ポンドの相場）、株価および金利の変動\n・金融市場における資金調達環境の変動および金融サービスにおける競争激化\n・効果的な販売・流通を実施する当社の能力\n・経営陣が設定したレベル、またはタイミングどおりに生産効率の実現と設備投資を実施する当社の能力\n・当社が営業活動を行っている市場内における法律、規制および政府政策の変更で、特にリコール等改善措置を含む安全性、貿易、環境保全、自動車排出ガス、\n燃費効率の面などにおいて当社の自動車事業に影響を与えるもの、または現在・将来の訴訟やその他の法的手続きの結果を含めた当社の その他の営業活動に\n影響を与える法律、規制および政府政策の変更など\n・当社が営業活動を行っている市場内における政治的および経済的な不安定さ\n・タイムリーに顧客のニーズに対応した新商品を開発し、それらが市場で受け入れられるようにする 当社の能力\n・ブランド・イメージの毀損\n・仕入先への部品供給の依存\n・原材料価格の上昇\n・デジタル情報技術 および情報セキュリティ への依存\n・当社が材料、部品、資材などを調達し、自社製品を製造、流通、販売する主な市場における、燃料供給の不足、電力・交通機能の マヒ、ストライキ、作業の中断、\nまたは労働力確保が中断されたり、 困難である状況など\n・生産および販売面への影響を含む、自然災害および感染症の発生・蔓延 、不安定な政治・経済、燃料供給の不足、社会基盤の障害、戦争、テロ、 ストライキ\nなどによる様々な影響\n・気候変動および低炭素経済への移行の影響\n以上の要素およびその他の変動要素全般に関する詳細については、当社の有価証券報告書 または米国証券取引委員会に提出された年次報告書（フォーム 20-F）を\nご参照ください。\n－補足3－" metadata={'source': 'Toyota_2023.pdf', 'page': 27}